content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- zparnold
+title: Overview of Cloud Native Security
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Kubernetes Security (and security in general) is an immense topic that has many
```

Comment:
- Is `Kubernetes Security` a concept in itself?
- The comment in parens does not add much
- immense is not a great descriptor

Reply:
@kbhawkey what would you suggest as a better descriptor?

Reply:
- Kubernetes security vs Kubernetes Security
- We know security is a big, important, complicated topic. What makes security challenging with cloud native? Since this is a single page, I think it would be helpful to declare/define the categories most important to k8s security -- you begin to do that with the 4C diagram. What does k8s offer today, by way of security? What else is needed?

There have been a number of blog posts about security, 
https://www.cncf.io/blog/2019/01/14/9-kubernetes-security-best-practices-everyone-must-follow/